sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientIntegrationTest.java

```diff
@@ -100,91 +97,12 @@ class ServiceBusAdministrationAsyncClientIntegrationTest extends TestProxyTestBa
         TEST_PROXY_REQUEST_MATCHERS = Collections.singletonList(customMatcher);
     }
 
+    private final AtomicReference<TokenCredential> credentialCached = new AtomicReference<>();
+
     public static Stream<Arguments> createHttpClients() {
         return Stream.of(Arguments.of(new NettyAsyncHttpClientBuilder().build()));
     }
 
-    /**
```

Comment:
These are valid use cases, we should keep them around.

Reply:
We moved this AzureSasCredential
(which is not federated) test to a new test file [ServiceBusNonFederatedIntegrationTest](https://github.com/Azure/azure-sdk-for-java/blob/0dd3a66efc5a988282d6e183ff0a9e05d94ca85a/sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusNonFederatedIntegrationTest.java)